sharedbuffers/mapped_struct.py

```diff
@@ -3267,6 +3306,20 @@ def _buffer_proxy_set(self, obj, code, elem):
         if obj is not None and not (obj.none_bitmap & self.mask):
             struct.pack_into(code, obj.buf, obj.offs + self.offs, elem)
 
+    def _buffer_proxy_cas(self, obj, code, exp_val, new_val):
+        # XXX: This is not atomic!
+        if obj is not None and not (obj.none_bitmap & self.mask):
+            tmp = struct.unpack_from(code, obj.buf, obj.offs + self.offs)[0]
+            if tmp == exp_val:
+                struct.pack_into(code, obj.buf, obj.offs + self.offs, new_val)
+                return True
+            return False
```

Comment:
This `return False` should be dedented one level I think (all similar ones in fact)